deb.go

```diff
@@ -20,19 +21,25 @@ type DebPkg struct {
 	err          error
 }
 
-// New creates new debian package
-func New() *DebPkg {
+// New creates new debian package, optionally provide an tempdir to write
+//  intermediate files, otherwise os.TempDir is used.
+func New(tempDir ...string) *DebPkg {
```

Comment:
Good compat, much better then the tempdir builtin which was to complicated. When tempdir doesn't exist. It is not automaticly created, maybe this is a nice feature to have (e.g build pipeline with custom tempdir which doesn't exist).

Reply:
I don't like the automatic directory-creation. It shouldn't be a job of debpkg, perhaps I can document it.